pkg/common/model.go

```diff
@@ -101,3 +108,136 @@ type Annotation struct {
 	Key   string `json:"key"`
 	Value string `json:"value"`
 }
+
+type DerivedImageList struct {
```

Comment:
Embed `PaginatedImagesResult` instead of existing attributes, and use `json:"DerivedImageList"`.